.buildkite/scripts/publish.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common.sh
+
+MAKEFILE=${1}
+
+check_is_arm
+add_bin_path
+make -C go -f "${MAKEFILE}" push"${is_arm}"
```

Comment:
In Jenkins, this command is retried 3 times in case it fails.
Should we add this ? 

```suggestion
retry 3 make -C go -f "${MAKEFILE}" push"${is_arm}"
```